src/Framework/IEventSource.cs

```diff
@@ -151,5 +156,10 @@ public interface IEventSource
         /// this event is raised to log any build event
         /// </summary>
         event AnyEventHandler AnyEventRaised;
+
+        /// <summary>
+        /// this event is raised to when telemetry is sent.
```

Comment:
Maybe "when telemetry events are created" or something? Since it might not be **sent**.